Brokerages/InteractiveBrokers/InteractiveBrokersBrokerage.cs

```diff
@@ -2479,7 +2479,7 @@ private bool CanSubscribe(Symbol symbol)
 
             return
                 (securityType == SecurityType.Equity && market == Market.USA) ||
-                (securityType == SecurityType.Forex && market == Market.FXCM) ||
+                (securityType == SecurityType.Forex && market == Market.Oanda) ||
                 (securityType == SecurityType.Option && market == Market.USA) ||
                 (securityType == SecurityType.Future);
```

Comment:
Maybe to avoid the duplication, could we use the `InteractiveBrokersBrokerageModel.DefaultMarketMap` directly for these checks and the one above (L2176)?

Reply:
Updated `MapSymbol` to reuse the `DefaultMarketMap`, but I would leave `Subscribe` as is for a few reasons:
- CFD trading with IB is not officially supported/tested
- With futures we are currently allowing all markets (will probably have to get back to this when we'll start addressing international markets)
- `Subscribe` is only called when deploying with the IB feed (but not with the QC feed)